sys/vm/vm_cheri_revoke.c

```diff
@@ -873,6 +873,31 @@ vm_cheri_revoke_pass(const struct vm_cheri_revoke_cookie *crc, int flags)
 	return res;
 }
 
+void
+vm_cheri_assert_consistent_clg(struct vm_map *map)
+{
+	pmap_t pmap = map->pmap;
+	vm_map_entry_t entry;
+	vm_offset_t addr;
+
+	/* Called with map lock held */
+
+	entry = vm_map_entry_first(map);
+
+	for (; entry != &map->header; entry = vm_map_entry_succ(entry))
+	{
```

Comment:
```suggestion
	for (; entry != &map->header; entry = vm_map_entry_succ(entry)) {
```

Reply:
I'll push a switch to VM_MAP_ENTRY_FOREACH once the current round of CI finishes.